pkg/reconciler/apiserversource/apiserversource.go

```diff
@@ -167,9 +162,6 @@ func (r *Reconciler) reconcile(ctx context.Context, source *v1alpha1.ApiServerSo
 	}
 	// Update source status
 	source.Status.PropagateDeploymentAvailability(ra)
-	if err = track(utils.ObjectRef(ra, deploymentGVK)); err != nil {
```

Comment:
why are we no longer tracking the deployment?

Reply:
Because we already had an event handler on it, and we own it. 

```	
deploymentInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
		FilterFunc: controller.Filter(v1alpha1.SchemeGroupVersion.WithKind("ApiServerSource")),
		Handler:    controller.HandleAll(impl.EnqueueControllerOf),
	})
```